src/pages/Autolending/LendTimingDropdown.vue

```diff
@@ -75,6 +80,30 @@ export default {
 			let notice = `Notice: Your previous setting was '${legacyAutoLendDescription}'`;
 			notice += this.changedTiming ? '.' : ', the closest matching setting above will be applied upon save.';
 			return notice;
+		},
+		autolendExplanationText() {
+			const now = Date.now();
+			const idleStartTime = Date.parse(this.cIdleStartTime);
+			const daysIdle = differenceInCalendarDays(now, idleStartTime);
+			const daysUntilLend = this.lendAfterDaysIdle - daysIdle;
+			const userBalance = parseFloat(this.userBalance).toFixed(2);
+			const loanAndDonationAmount = ((1 + this.donationPercentage / 100) * 25).toFixed(2);
```

Comment:
You shouldn't use `toFixed` for the userBalance and the loanAndDonationAmount because it's turning them into strings instead of numbers, which results in them being compared alphabetically rather than numerically. For instance, `5 > 26 === false`, but `'5' > '26' === true` because when compared as strings they are compared character by character, and 5 is bigger than 2, so string '5' is therefore bigger than string '26'

Reply:
Changed to use numeral, which on all accounts is much much better.

Reply:
Using numeral's `format` function still gives you the same problem though, as that's returning string and not numbers.

Reply:
The problem is that you want to use `userBalance` and `loanAndDonationAmount` as numbers when you are comparing them (so that you don't end up with '26' being considered smaller than '5') but you also want to use them as formatted strings in the messages that you print out so that they look good. You can make a test that reveals this problem by setting an non-null cIdleStartTime and a balance that is greater than 3 and less than 25. In those conditions, you would just expect the message to contain 'Your current balance is lower', but when you run the test you'll find that it's actually saying that the balance is eligible

Reply:
One way you could fix it is by using different variables for the numbers and the formatted strings
```
const userBalance = numeral(this.userBalance).value();
const userBalanceFormatted = numeral(this.userBalance).format('0,0.00');
const loanAndDonationAmount = (1 + this.donationPercentage / 100) * 25;
const loanAndDonationAmountFormatted = numeral(loanAndDonationAmount).format('0,0.00');
```
That allows `userBalance` and `loanAndDonationAmount` to be safely used in comparisons (like `if(userBalance < loanAndDonationAmount)`) and then `userBalanceFormatted` and `loanAndDonationAmountFormatted` can be used in the template strings for the message